docs/source/api/apollo-server.md

```diff
@@ -480,6 +480,17 @@ addMockFunctionsToSchema({
 
    A human-readable name for the variant of a schema (i.e. staging, EU). Setting this value will cause metrics to be segmented in the Apollo Graph Manager UI. Additionally schema validation with a graph variant will only check metrics associated with the same string.
 
+* `traceReporting`: Boolean | async (GraphQLRequestContextDidResolveOperation) => Boolean
+
+    If this option returns `true`, tracing and instrumentation will be enabled for a
```

Comment:
The wording here is a bit odd, around "returns `true`". Maybe remix this whole thing:

Specify whether to report traces and metrics to Apollo Graph Manager. This may be specified as a boolean or as an async function returning a (Promise resolving to a) boolean. The function will receive a `GraphQLRequestContextDidResolveOperation` with client and operation information allowing the choice of whether to include a given request in trace and metric reporting to be made on a per-request basis. The default value is `true`.